crates/context/config/src/client/env/proxy/query/proposal_approvers.rs

```diff
@@ -16,29 +15,27 @@ pub(super) struct ProposalApproversRequest {
 impl Method<Near> for ProposalApproversRequest {
     const METHOD: &'static str = "get_proposal_approvers";
 
-    type Returns = Vec<User>;
+    type Returns = Vec<ContextIdentity>;
 
     fn encode(self) -> eyre::Result<Vec<u8>> {
         serde_json::to_vec(&self).map_err(Into::into)
     }
 
     fn decode(response: Vec<u8>) -> eyre::Result<Self::Returns> {
-        let signers: Option<Vec<Repr<SignerId>>> = serde_json::from_slice(&response)?;
+        let signers: Option<Vec<Repr<ContextIdentity>>> = serde_json::from_slice(&response)?;
 
         Ok(signers
             .unwrap_or_default()
             .into_iter()
-            .map(|signer_id| User {
-                identity_public_key: signer_id.to_string(),
-            })
+            .map(|signer_id| signer_id.rt().expect("infallible conversion"))
             .collect())
```

Comment:
we can reuse the allocation, since `Repr` is `#[repr(transparent)]`

example:

https://github.com/calimero-network/core/blob/d303ad19ccca83360b43b112fcbf3daac37de4f9/crates/context/config/src/client/env/config/query/members.rs#L30-L36